src/Pages/StatusPage/Create/index.jsx

```
@@ -87,30 +87,31 @@ const CreateStatusPage = () => {
 
-	const handleImageChange = useCallback((event) => {
-		const img = event.target?.files?.[0];
-		const newLogo = {
-			src: URL.createObjectURL(img),
-			name: img.name,
-			type: img.type,
-			size: img.size,
-		};
+	const handleImageChange = useCallback((fileObj) => {
+		if (!fileObj || !fileObj.file) return;
+	  
```

ESLint

[error] Mixed spaces and tabs.

(no-mixed-spaces-and-tabs)